en/technical-reference/going-live.md

```diff
@@ -195,8 +202,8 @@ Your website is now integrated with Midtrans. For more information, refer to [Co
 3. Enter **name** this text appears on the button displayed to the customer.
 4. Click **Midtrans** *Plugin* option button.
 5. Click **Production** *Mode* option button.
-6. Enter **Server key** with the *Server Key* retrieved from [Step 3 above](#_3-retrieving-server-key-and-client-key).
-7. Enter **Client key** with the *Client Key* retrieved from [Step 3 above](#_3-retrieving-server-key-and-client-key).
+6. Enter **Server key** with the *Server Key* retrieved from [Step 2 above](#_2-retrieving-production-server-key-and-client-key).
+7. Enter **Client key** with the *Client Key* retrieved from [Step 2 above](#_2-retrieving-production-server-key-and-client-key).
 8. Click **Save**.
 
```

Comment:
Also please replace contact email on bottom of this page with Contact Us web page URL